dgraph/cmd/alpha/run_test.go

```diff
@@ -1471,6 +1471,31 @@ func TestIPStringParsing(t *testing.T) {
 	require.NoError(t, err)
 	require.NotEqual(t, net.IPv6zero, addrRange[0].Lower)
 	require.NotEqual(t, addrRange[0].Lower, addrRange[0].Upper)
+
+	addrRange, err = getIPsFromString("")
```

Comment:
Nit: This sort of replication is a good candidate for "table driven tests"; making the test cases easier to read and allowing parallel execution of tests. No need to change this, just an FYI...